docs/getting-started.rst

```diff
@@ -152,3 +152,9 @@ how you could identify yourself:
   integration_name=MyScript/0.9.0
 
 See the :doc:`authentication` documentation for more information about credentials.
+
+Next Steps
+----------
+
+ - :doc:`concepts`: General information about retrieving data from your Carbon Black Cloud instance
```

Comment:
I would rephrase concepts more towards the nuance of using the CBC SDK as it describes the python constructs more than the Carbon Black Cloud data